gcp/table_gcp_bigquery_table.go

```diff
@@ -0,0 +1,217 @@
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Id"),
+			},
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Labels"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(gcpBigqueryTableAkas),
+			},
+			// standard gcp columns
+			{
+				Name:        "location",
+				Description: ColumnDescriptionLocation,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getBigqueryTable,
+				Transform:   transform.FromField("Location"),
+			},
+			{
+				Name:        "project",
+				Description: ColumnDescriptionProject,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("TableReference.ProjectId"),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+func listBigqueryTables(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("ListBigqueryTables")
+	// Get a details of Cloud Dataset
+	dataset := h.Item.(*bigquery.DatasetListDatasets)
+	// Create Service Connection
+	service, err := BigQueryService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+	resp := service.Tables.List(project, dataset.DatasetReference.DatasetId)
+	if err := resp.Pages(ctx, func(page *bigquery.TableList) error {
+		for _, table := range page.Tables {
+			d.StreamListItem(ctx, table)
+		}
+		return nil
+	}); err != nil {
+		return nil, err
+	}
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+func getBigqueryTable(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getBigqueryTable")
+	// Create Service Connection
+	service, err := BigQueryService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+	var datasetID, id string
+	if h.Item != nil {
+		data := h.Item.(*bigquery.TableListTables)
+		datasetID = data.TableReference.DatasetId
+		id = data.TableReference.TableId
+	} else {
+		datasetID = d.KeyColumnQuals["dataset_id"].GetStringValue()
+		id = d.KeyColumnQuals["id"].GetStringValue()
+	}
+	resp, err := service.Tables.Get(project, datasetID, id).Do()
+	if err != nil {
+		return nil, err
+	}
+	return resp, err
+}
+
+//// TRANSFORM FUNCTIONS
+func gcpBigqueryTableAkas(ctx context.Context, h *transform.TransformData) (interface{}, error) {
+	data := TableID(h.HydrateItem)
+	projectID := strings.Split(data, ":")[0]
+	id := strings.Split(data, ":")[1]
+	akas := []string{"gcp://bigquery.googleapis.com/projects/" + projectID + "/tables/" + id}
```

Comment:
Can we include dataset details in `akas`?
e.g. `gcp://bigquery.googleapis.com/projects/<project_id>/datasets/<dataset_id>/tables/<table_id>`